### PR TITLE
TST: Add unit tests for PyDMImageView

### PR DESCRIPTION
**Pull request by hmbui**
Although the coverage report states that lines 33-72 were not executed, they were, but in a separate thread (ImageUpdateThread).

### CODE REVIEW

File: pydm/widgets/image.py [hunk 1]
```diff
@@ -272,7 +272,9 @@ def colorMap(self, new_cmap):
         new_cmap : PyDMColorMap
         """
         self._colormap = new_cmap
-        self._cm_colors = self.color_maps[new_cmap]
+        if new_cmap:
```
Comment (hhslepicka) [inline pydm/widgets/image.py]: Why not also wrap the ``self._colormap`` with this ``if`` statement?

File: pydm/widgets/image.py [hunk 2]
```diff
@@ -344,8 +346,6 @@ def image_width_changed(self, new_width):
         new_width : int
             The new image width
         """
-        if new_width is None:
```
Comment (hhslepicka) [inline pydm/widgets/image.py]: Why was this removed?

Comment (hmbui) [reply pydm/widgets/image.py]: The slot only expects an int value. If new_width is None, an exception will be raised, so there's no need for a None check here.

File: pydm/widgets/image.py (hunk 1 above)
Comment (hmbui) [reply pydm/widgets/image.py]: Do you mean checking for self.color_maps?

if new_cmap and self.color_maps:
...

File: pydm/widgets/image.py (hunk 2 above)
Comment (hhslepicka) [reply pydm/widgets/image.py]: We should never allow the code to drag down the whole application.
This code can also be invoked from command line and one can end up sending `None` down the pipe and crash if we don't protect with the proper check.
Please add again the line that was removed.

File: pydm/widgets/image.py (hunk 1 above)
Comment (hhslepicka) [reply pydm/widgets/image.py]: I mean make it:
```
if new_cmap:
    self._colormap = new_cmap
    ...
```
Otherwise self._colormap can be `None`.
